### ISSUE TITLE
Further encapsulation

### ISSUE THREAD

**Issue by herzinger** [inuitcss/inuitcss#178]
I was about to write some examples in codepen for the issue #177, and as discussed in issue #169, it's kinda of a pain to include inuitcss for this kind of stuff. One could come here and grab the raw link of only the files the snippet will need, or copy-paste the necessary code, but it's still a lot of stuff in some cases. The project I'm working right now is a UI toolkit built on top of inuitcss, and to make it easier for the people who'll have to use it, I further grouped the files with `@imports`, like so:

```
ui-toolkit
|
|- _core (settings and tools layers)
|- _base (generic and elements layers)
|- _objects
|- _components
|- _utilities
```
On top of each file I wrote a little documentation and reference links about the layers present, and I also have my table of contents divided in them. This makes it easy to import the whole toolkit and still keep the layers and source order organized, even when among the project's specific partials. Something like that would make it a lot easier to import files for codepen and the likes, as long as they support raw sass files.

**Comment by herzinger** [comment]
Or not... After a little testing, codepen doesn't work well with `@imports` in external files. And after a second tough, if it was the case, I could use the `example.main.scss` raw. Still, the model above has been really useful to me, inclusive making it easier to sell the whole methodology to my team and lessem the learning curve.

**Comment by csshugs** [comment]
@herzinger 
The aim of inuitcss is being a really generic CSS framework for use in projects. 
Providing something like what you proposed above would mean we have to change our file architecture. But for using inuitcss as a projects CSS framework, there is no need to change that. In fact we are very happy with our ITCSS structure. inuitcss is not repsonsible for being a framework which works well in codepen etc. and we will definitely not make compromises for such cases. 

Anyone can encapsulate the layers further as they like, just like you did, in their main.scss file. This is not something inuitcss has/wants to take care of.

The question of serving a generated CSS file of the whole framework which could be used for demo purposes (e.g. on codepen) is being addressed in #169.

**Comment by herzinger** [comment]
I was not talking about changing the ITCSS structure, just adding some barrels. But yeah, it's easy enough to do at a project level. Thanks for the answer.
